### PR TITLE
Contact validation with JavaScript

### PR DESCRIPTION
**Pull request by chibunduonyeje1**
### The project was built with the following:

- Add error message field to the contact form.
- Used CSS to style the error message field and add text color red.
- Used JavaScript to the input field and throw an error message if the email is in Uppercase.
- Used Linters to check for error

### CODE REVIEW

File: assets/js/validator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  } else {
+    nameErrorMsg.innerHTML = '';
+  }
+
+  if (emailField.value.length === 0) {
+    emailErrorMsg.innerHTML = 'Your Email is required';
+  } else if (emailField.value !== emailField.value.toLowerCase()) {
+    emailErrorMsg.innerHTML = 'Please input your email address in lowercase.';
+  } else {
+    emailErrorMsg.innerHTML = '';
```
Comment (algerina) [inline assets/js/validator.js]: - [ ] Well done checking if the email is all lowercase by comparing if the email and it's lowercase converted version are the same using` email === email.toLowerCase()`, however when I tried it with an email with Uppercase, I've got the error message as required but the form was submitted normally. Kindly try to fix the issue, if the email is in Uppercase the form should not be submitted.

**Review by algerina — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

### Hello @chibunduonyeje1,
 ✋🏼 Nice done I am impressed. Your project is great. There are just some small changes to add.

HIGHLIGHTS
- Your Project looks great :heavy_check_mark: 
- No Linter errors :heavy_check_mark: 
- You have a descriptive PR :heavy_check_mark:   

## Required changes :

- [ ] Please, check the comments under the review.

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: assets/js/validator.js (hunk 1 above)
Comment (Whoistolu) [reply assets/js/validator.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @chibunduonyeje1 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**